eq-author/cypress/integration/authenticated/builder_spec.js

```diff
@@ -640,13 +640,13 @@ describe("builder", () => {
       }
     };
 
-    it('should add an "other" answer', () => {
+    it("should add an additionalValue option answer", () => {
       addMultipleChoiceAnswer();
       cy.get(testId("option-label")).should("have.length", 2);
       cy.get(testId("other-answer")).should("have.length", 1);
```

Comment:
Can we rename these from `other-answer` to `additional-answer`

Reply:
Yeah, wasn't really sure where to draw the line here. From a UI perspective  it will still be an "other" answer, least until/if we get a new design so was a bit unsure on whether the labels and the other attribs should be changed or not?

Reply:
I worry about new people not knowing the history and getting confused by this inconsistency.

Reply:
As discussed there is a redesign of the UI to support different answer types for this as well so we look at fully removing "Other" later.